fft/src/KokkosFFT_Plans.hpp

```diff
@@ -39,34 +45,72 @@ namespace Impl {
 template <typename ExecutionSpace, typename InViewType, typename OutViewType,
           std::size_t DIM = 1>
 class Plan {
-  using execSpace      = ExecutionSpace;
-  using in_value_type  = typename InViewType::non_const_value_type;
+  //! The type of Kokkos execution pace
+  using execSpace = ExecutionSpace;
+
+  //! The value type of input view
+  using in_value_type = typename InViewType::non_const_value_type;
+
+  //! The value type of output view
   using out_value_type = typename OutViewType::non_const_value_type;
-  using float_type     = KokkosFFT::Impl::real_type_t<in_value_type>;
+
+  //! The real value type of input/output views
+  using float_type = KokkosFFT::Impl::real_type_t<in_value_type>;
+
+  //! The type of fft plan
   using fft_plan_type =
       typename KokkosFFT::Impl::FFTPlanType<ExecutionSpace, in_value_type,
                                             out_value_type>::type;
-  using fft_size_type       = std::size_t;
-  using map_type            = axis_type<InViewType::rank()>;
-  using nonConstInViewType  = std::remove_cv_t<InViewType>;
+
+  //! The type of fft size
+  using fft_size_type = std::size_t;
+
+  //! The type of map for transpose
+  using map_type = axis_type<InViewType::rank()>;
+
+  //! The non-const View type of input view
+  using nonConstInViewType = std::remove_cv_t<InViewType>;
+
+  //! The non-const View type of output view
   using nonConstOutViewType = std::remove_cv_t<OutViewType>;
-  using extents_type        = shape_type<InViewType::rank()>;
 
+  //! The type of extents of input/output views
+  using extents_type = shape_type<InViewType::rank()>;
+
+  //! Dynamically allocatable fft plan.
   std::unique_ptr<fft_plan_type> m_plan;
+
+  //! fft size
   fft_size_type m_fft_size;
+
+  //! maps for forward and backward transpose
   map_type m_map, m_map_inv;
+
+  //! whether transpose is needed or not
   bool m_is_transpose_needed;
+
+  //! axes for fft
   axis_type<DIM> m_axes;
+
+  //! directions of fft
   KokkosFFT::Impl::Direction m_direction;
 
-  // Keep extents
+  //! extents of input/output views
   extents_type m_in_extents, m_out_extents;
 
-  // Only used when transpose needed
+  //! Internal buffers used for transpose
```

Comment:
without `@{` `@}`, you only comment the first next variable `m_in_T`, not the following one `m_out_T`

Reply:
So, we need the comment like this?

`//@{ //! Internal buffers used for transpose @}//`

Reply:
not exactly, the `@{` and `@}` should surround the variables, see https://www.doxygen.nl/manual/grouping.html

Reply:
So, 

```
///@{ 
  //! extents of input/output views
  extents_type m_in_extents, m_out_extents;
///@}
```